### ISSUE TITLE
1.19Maven Problem

### ISSUE THREAD

**Issue by kopolinka32** [dmulloy2/ProtocolLib#1720]
- [ ] This issue is not solved in a development build

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Version Info**
Provide your ProtocolLib install info with `/protocol dump` through pastebin.

**Additional context**

Caused by: java.lang.NullPointerException: Cannot invoke "com.comphenix.protocol.ProtocolManager.createPacket(com.comphenix.protocol.PacketType)" because "protocolManager" is null
        at com.challangerson.guilds.element.tablist.TabManager.sendTabList(TabManager.java:39) ~[?:?]
        at com.challangerson.guilds.listeners.PlayerJoinListener.onPlayerJoinListener(PlayerJoinListener.java:22) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.19-R0.1-SNAPSHOT.jar:?]
        ... 15 more

After adding a 1.19 maven cant accept 

        <dependency>
            <groupId>com.comphenix.protocol</groupId>
            <artifactId>ProtocolLib</artifactId>
            <version>5.0.0-SNAPSHOT-B573</version>
        </dependency>

how can i add a newest version to maven or my respositories the max is 5.0.0

**Comment by derklaro** [comment]
Make sure you set the scope of your dependency to provided like:
```xml
    <dependency>
        <groupId>com.comphenix.protocol</groupId>
        <artifactId>ProtocolLib</artifactId>
        <version>5.0.0-SNAPSHOT-B573</version>
        <scope>provided</scope>
    </dependency>
```

**Comment by kopolinka32** [comment]
> Make sure you set the scope of your dependency to provided like:
> 
> ```
>     <dependency>
>         <groupId>com.comphenix.protocol</groupId>
>         <artifactId>ProtocolLib</artifactId>
>         <version>5.0.0-SNAPSHOT-B573</version>
>         <scope>provided</scope>
>     </dependency>
> ```

Not working actually

**Comment by derklaro** [comment]
remove the -b573 suffix

**Comment by kopolinka32** [comment]
> 

Hi i have a problem i have something like that :

    @SneakyThrows
    public void sendTabList(Player player) {

        ProtocolManager protocolManager = ProtocolLibrary.getProtocolManager();

        MinecraftServer server = ((CraftServer)Bukkit.getServer()).getServer();
        WorldServer world = ((CraftWorld)Bukkit.getWorlds().get(0)).getHandle();
        GameProfile gameProfile = new GameProfile(UUID.randomUUID(), "");


        EntityPlayer entityPlayer = new EntityPlayer(server, world, gameProfile, null);

        PacketContainer packetContainer = protocolManager.createPacket(PacketType.Play.Server.PLAYER_INFO);
        packetContainer.getPlayerInfoAction().write(0, EnumWrappers.PlayerInfoAction.ADD_PLAYER);
        packetContainer.getModifier().write(1, entityPlayer);

        protocolManager.sendServerPacket(player, packetContainer);

    }

Andd a problem is : https://pastebin.com/g23Rzdbq

**Comment by derklaro** [comment]
This issue can be closed now, see #1726

**Comment by UnknownKriodluk** [comment]
Hi! I have an error: Could not find artifact com.comphenix.protocol:ProtocolLib:pom:5.0.0-SNAPSHOT-B573 in papermc-repo (https://repo.papermc.io/repository/maven-public/)
